components/card/card-footer-link.js

```diff
@@ -61,7 +61,7 @@ class CardFooterLink extends FocusMixin(RtlMixin(LitElement)) {
 			 */
 			target: { type: String, reflect: true },
 			/**
-			 * REQUIRED: Accessible text for the link (not visible, gets announced when user focuses)
+			 * ACCESSIBILITY: REQUIRED: Accessible text for the link for screen reader users
```

Comment:
maybe something like
```suggestion
			 * ACCESSIBILITY: REQUIRED: Accessible text describing the link icon to screen reader users
```
I'm not sure if "describing" implies longer text than we'd want though. Depends how we see this used

Reply:
I'd also worry a bit that they'd _actually_ describe the icon -- i.e. "A pencil icon" instead of its purpose ("Edit quiz"). Could we just call this the "Accessible text for the link", similar to `<d2l-button-icon>`'s `text` ("Accessible text for the button").

Reply:
Yeah, I had the same thought as Dave, I didn't want to risk people describing what the icon looks like, which is why I shied away for using the word `describe`. I guess I could try doing:
```suggestion
			 * ACCESSIBILITY: REQUIRED: Accessible text describing the purpose of the link to screen reader users
```

Reply:
Dave's suggestion is short and sweet, and we could expand on it in an Accessibility section since we also want to explain how/why it's only required when the card is clickable (has an `href`)?